hta/utils/utils.py

```diff
@@ -79,23 +79,31 @@ def is_memory_kernel(name: str) -> bool:
 
 
 def is_computer_kernel(name: str) -> bool:
+    """
+    Will depracate soon. Use is_compute_kernel() instead.
+    """
+    non_computer_kernel_re = re.compile(r"(^ncclKernel)|(.*(Memcpy)|(Memset))|(.*Sync)")
+    return not non_computer_kernel_re.match(name)
```

Comment:
Why not just call `return is_compute_kernel(name)` and add a `deprecated` decorator